### ISSUE TITLE
can't re-use a helm buffer

### ISSUE THREAD

**Issue by denysos** [emacs-helm/helm#1517]
Hi,
I have the following message in the "message" buffer when I try to re-use a "helm" buffer (most of the time "helm-find-file") :

`Error during redisplay: (eval (format "L%-3d" (helm-candidate-number-at-point))) signaled (error "No buffer named *helm*")`

in fact moving inside the buffer is strange, and no keys work.

at the beginning I thought that it was a init-file problem because I am new to Emacs or a "Windows" related bug.

but I tried to install an "out of the box" version like "super-emacs" in an Ubuntu Virtual Machine,
in fact there is nothing to add in your init file to start using super-Emacs ...

today I use helm version 20160524.820 and the problem is still here.

is it a good idea to get back to an "old" helm-find-file buffer ?
or should I start with a new one with C-x C-f ?

thank you

**Comment by thierryvolpiatto** [comment]
denysos <notifications@github.com> writes:

> is it a good idea to get back to an "old" helm-find-file buffer ?

No, it is a very bad idea.

> or should I start with a new one with C-x C-f ?

Yes, please.

-- 
Thierry

**Comment by thierryvolpiatto** [comment]
>I have the following message in the "message" buffer when I try to re-use a "helm" buffer (most of >the time "helm-find-file") :

```
Error during redisplay: (eval (format "L%-3d" (helm-candidate-number-at-point))) signaled (error "No buffer named *helm*")
```
Should be fixed now, though you have to understand these buffers are here for next usage of helm and normally you have nothing to do inside these buffers, to revive a helm-buffer use M-x helm-resume.

**Comment by denysos** [comment]
thank you for your incredible speed
I got lost by the error message

I tried to go on with a helm buffer just by show it again on the screen,
because I learned to navigate among buffers ... and it seemed to be
natural... but it was not the good thing to do.

now I use "(C-u) C-x c b" and it works fine

thank you

2016-05-26 15:34 GMT+02:00 Thierry Volpiatto <notifications@github.com>:

> Closed #1517 <https://github.com/emacs-helm/helm/issues/1517>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/emacs-helm/helm/issues/1517#event-673166147>
>



-- 
Denis Mourier
06-99-20-67-88
retrouvez-moi sur Viadeo <http://www.viadeo.com/fr/profile/denis.mourier>
retrouvez-moi sur LinkedIn <http://fr.linkedin.com/in/denismourier>

**Comment by nunowallenstein** [comment]
I get this same bug when i try to change the buffer with Mouse 1 at the title of the buffer in the lower left part of the buffer. 
When i for example, find a file (C-x C-f) and change the buffer with the mouse 1, i end up in a buffer called \***helm find files**\* and i cannot switch to other buffers this way, after encountering this buffer. This also happens every time i encounter a \***helm buffer**\* this way,  preventing me from switching buffer using mouse 1. On the buffer \***Messages**\* these kind of error messages occur. How can i fix them?

```
Error during redisplay: (eval (with-helm-buffer (helm-show-candidate-number (car-safe helm-mode-line-string)))) signaled (error "No buffer named *helm*") [11 times]
helm-display-mode-line: Wrong type argument: stringp, ("File(s)" "\\<helm-map>\\[helm-help]:Help \\[helm-select-action]:Act \\[helm-maybe-exit-minibuffer]/f1/f2/f-n:NthAct \\[helm-toggle-suspend-update]:Tog.suspend \\[helm-customize-group]:Conf")Error during redisplay: (eval (with-helm-buffer (helm-show-candidate-number (car-safe helm-mode-line-string)))) signaled (error "No buffer named *helm*")
Error during redisplay: (eval (with-helm-buffer (helm-show-candidate-number (car-safe helm-mode-line-string)))) signaled (error "No buffer named *helm*") [9 times]
auto trimmed 0 chars
helm-display-mode-line: Wrong type argument: stringp, ("File(s)" "\\<helm-map>\\[helm-help]:Help \\[helm-select-action]:Act \\[helm-maybe-exit-minibuffer]/f1/f2/f-n:NthAct \\[helm-toggle-suspend-update]:Tog.suspend \\[helm-customize-group]:Conf") [2 times]
next-line: End of buffer
Quit
QuitError during redisplay: (eval (with-helm-buffer (helm-show-candidate-number (car-safe helm-mode-line-string)))) signaled (error "No buffer named *helm*")
Error during redisplay: (eval (with-helm-buffer (helm-show-candidate-number (car-safe helm-mode-line-string)))) signaled (error "No buffer named *helm*")

```
